src/Raven.Server/Routing/RouteInformation.cs

```diff
@@ -86,6 +87,28 @@ public Task CreateDatabase(RequestHandlerContext context)
                 throw new NodeIsPassiveException($"Can't perform actions on the database '{databaseName}' while the node is passive.");
             }
 
+            if (context.RavenServer.ServerStore.IdleDatabases.Count > 0 
```

Comment:
Why check the count? Just use `TryGetValue`

Reply:
done